signature/sigstore/fulcio/no_fulcio.go

```diff
@@ -0,0 +1,45 @@
+//go:build containers_image_fulcio_stub
```

Comment:
Absolutely non-blocking: I would weakly prefer these files to be named `fulcio_stub.go` and `rekor_stub.go`.